src/main/java/com/google/devtools/build/lib/buildtool/ExecutionTool.java

```diff
@@ -937,10 +938,14 @@ private Builder createBuilder(
   public static void configureResourceManager(ResourceManager resourceMgr, BuildRequest request) {
     ExecutionOptions options = request.getOptions(ExecutionOptions.class);
     resourceMgr.setPrioritizeLocalActions(options.prioritizeLocalActions);
+    ImmutableMap<String, Float> extraResources = options.localExtraResources.stream().collect(
```

Comment:
If we have same entry multiple time then we return an error here. Let's do it more gracefully with more context.

Reply:
I can't tell what you mean here. If I specify the local resources twice I don't see a problem:
```
# Allow bazel to allocate tests to 1 cw310 board.
test --local_extra_resources=cw310=1
test --local_extra_resources=cw310=1
test --local_extra_resources=cw310=2
```
Seems to work fine

`(v1,v2) -> v1` is different from how most options are handled where values can't be overridden. I'll change that to: `(v1,v2) -> v2` so args can override rc files and the priorities apply properly.

Reply:
We could use min of v1 and v2 so that all restrictions are applied. PLMK

Reply:
I guess it's ok. 

I am not sure about importing another .rc files. In that case it's hidden that some files could change the value of `local_extra_resources `. 

@larsrc-google do we guarantee the all lines in .rc files are applied in the same order? E.g. what do we do if we have multiple `strategy=` flags with same mnemonic?